storage/types.go

```diff
@@ -381,9 +384,18 @@ type databaseShard interface {
 		flush persist.Flush,
 	) error
 
+	// Snapshot snapshot's the unflushed series' in this shard.
+	Snapshot(blockStart time.Time, snapshotStart time.Time, flush persist.Flush) error
```

Comment:
Can you make the function signature for the two Snapshot decls identical: 

```
// the first one uses
blockStart, snapshotTime time.Time

// the second one instead does 
blockStart time.Time, snapshotTime time.Time
```

Reply:
done